Simplenote/InterlinkViewController.swift

```diff
@@ -32,6 +32,10 @@ class InterlinkViewController: NSViewController {
         return ResultsController(viewContext: mainContext, sortedBy: [sortDescriptor])
     }()
 
+    /// Closure to be executed whenever a Note is selected. The Interlink URL will be passed along.
+    ///
+    var onInsertInterlink: ((String) -> Void)?
```

Comment:
This is me being a little picky, but could we include a mention that the callback is one-time use only?  If at some point in the future interlinks were ever used in a way where the owner didn't dismiss the view as part of the call back block the control becoming suddenly [partially] unresponsive might be surprising.  Hopefully a note about it would help a future coder.

Alternatively, would it make sense for the owner to be responsible for nullifying the callback since it handles the assignment?

Reply:
On second thoughts, let's drop the "reset callback" on selection (since the callback is weak anyways!!).

The ideal flow involves, as you mentioned, the owner that assigns the callback actually nullifying the closure. But that code doesn't look cute LOL 😂 !!